app/scripts/events.js

```diff
@@ -682,6 +709,15 @@ define([
         }
       });
 
+      $('.l-workspace-wrap').on('click', '.btn_input_send', function() {
```

Comment:
https://github.com/QuickBlox/CodeStyle-Guidelines-for-Developers/tree/master/web/js
Starts with prefix j- for variable which save HTML-DOM element;